Server content/Resources/Prototypes/Roles/Jobs/Engineering/builder.yml

```
@@ -0,0 +1,35 @@
+- type: job
+  id: Builder
+  name: job-name-builder
```

```suggestion
  name: Builder
```
Нету Американ локализации имени роли